Update to 2.087.0 stable release

ErnyTech opened the pull request:
https://dlang.org/changelog/2.087.0.html
https://github.com/dlang/dmd/releases/tag/v2.087.0

Other changes not related to DMD changelog:
- Removed the hack for the soname symlink

Commit:
Update to 2.087.0 stable release

https://dlang.org/changelog/2.087.0.html
https://github.com/dlang/dmd/releases/tag/v2.087.0

joseph-wakeling-frequenz commented:
You may want to rewrite the second commit message a bit, it looks like some content got mixed up there.

I would suggest also not having a `Revert Revert` in the title, but just (in the title) describe removing the hack for the soname symlink.  i.e. treat it as a fresh patch even if it is technically a revert ;-)

Commit:
Removed the hack for the soname symlink

Since the versions starting from phobos 2.087 have this
merged pull request: dlang/phobos#6966
It is no longer necessary to manually create the symlink.

ghost commented:
Ok done
